develop/learning-paths/mobile/articles/01-beginning-android-portal-development/02-build-mobile-sdk-android.markdown

```diff
@@ -6,7 +6,7 @@ have to build it? Can't I just use the pre-built Mobile SDK that Liferay already
 provides?" Fantastic question! The reason is that the pre-built Mobile SDK 
```

Comment:
It would be good to inform the user how to temporarily turn off Gradle Daemon when they want to run `createModule`. Something like this ...

The `createModule` task can't run properly if Gradle Daemon is running. To turn off the daemon, you can include  the switch `--no-daemon` in your command:
`./gradlew --no-daemon createModule`